src/DynamoCoreWpf/Properties/AssemblyInfo.cs

```diff
@@ -40,3 +40,4 @@
 [assembly: InternalsVisibleTo("WorkspaceDependencyViewExtension")]
 [assembly: InternalsVisibleTo("DocumentationBrowserViewExtension")]
 [assembly: InternalsVisibleTo("SystemTestServices")]
+[assembly: InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
@QilongTang what is this?

Reply:
@mjkkirschner Brought in Moq dependency by https://github.com/DynamoDS/Dynamo/pull/10569. Details can be found here as well https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Properties/AssemblyInfo.cs#L24

Reply:
That's correct. To be clear, that is the name of the assembly where mocks created using Moq are placed.